core/capabilities/ccip/ccipsolana/gas_helpers.go

```diff
@@ -0,0 +1,22 @@
+package ccipsolana
+
+import (
+	cciptypes "github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
+)
+
+func NewGasEstimateProvider() EstimateProvider {
+	return EstimateProvider{}
+}
+
+type EstimateProvider struct {
+}
+
+// CalculateMerkleTreeGas is not implemented
+func (gp EstimateProvider) CalculateMerkleTreeGas(numRequests int) uint64 {
+	return 0
```

Comment:
We need to check with @winder or others in CCIP team, if 0 is acceptable for the no-op here, or do we need a very large value.

Reply:
Checked with @winder , "`The estimate provider is used to determine if there is a gas spike. Returning 0 will ensure we don't reject a message due to a bad estimate."`